CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/startup/Startup.java

```diff
@@ -54,6 +54,8 @@ public void run() {
                 ? String.format("%s %s", BrandingUtilities.APPLICATION_NAME, environment)
                 : BrandingUtilities.APPLICATION_NAME;
 
+        ConstellationLAFSettings.applyTabColorSettings();
```

Comment:
As discovered in the unit testing, this should only be run when we are **not** in headless mode.
Will move this line into the if statement below.